flink-examples/src/main/java/io/pravega/examples/flink/alert/AccessLog.java

```diff
@@ -10,22 +10,30 @@
  */
 package io.pravega.examples.flink.alert;
 
-import io.pravega.shaded.com.google.gson.Gson;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.annotation.JsonProperty;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.core.JsonProcessingException;
+import org.apache.flink.shaded.jackson2.com.fasterxml.jackson.databind.ObjectMapper;
+import org.joda.time.DateTime;
 
 /**
  * Object to process Apache access log
  */
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Are we expecting any unknown fields to be part of the json data?

Reply:
Yes, there a quite a few more fields in the standard apache access log. I was also planning to use this for other examples, potentially.